riscv-asm.md

```diff
@@ -568,6 +585,48 @@ pseudoinstruction (`fl{h|w|d|q}`).
 	flw fa0, .VAL, t0
 ```
 
+
+<a name="fli-operand-value"></a>FLI operands reference table
+
+Value                       | Possible inputs values
```

Comment:
I suggest changing "Possible inputs values" to "Example legal input values".  This makes it clear these aren't the only possibilities.  (Also note that "input" should not be plural.)